My First Pull Request !

zmekonnen251 opened the pull request:
index.html, style.css files, and linter for local and remote repository added.

VanessaAoki commented on README.md:
```diff
@@ -14,8 +14,8 @@
-- Twitter: [@twitterhandle](https://twitter.com/mek_zela)
+- GitHub: [@zmekonnen251](https://github.com/zmekonnen251)
+- Twitter: [@mek_zela](https://twitter.com/mek_zela)
 - LinkedIn: [LinkedIn](www.linkedin.com/in/zelalem-g-mekonnen)
```
- [ ] [OPTIONAL]: Right now when I click on this icon it redirects me to a 404 page. To make your repo looks professional, you should double-check your links to make sure they all work! Also, consider changing the `[LinkedIn]` to your name, considering it already says that the following link will be your LinkedIn profile!

zmekonnen251 replied:
Thank you Vanessa Aoki very your comment and I have corrected it.

zmekonnen251 replied:
Thank you @VanessaAoki for your helpful suggestion. I modified it as you suggested.

VanessaAoki reviewed (changes requested):
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @zmekonnen251, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] Your README file looks very professional and the PR has a descriptive title :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
- [ ] One of the project requirements was to add linters to your project, you can check them out [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js). _You should consider adding local linters in the gitignore file, to keep your project clean!_
- [ ] Your Pull Request should have a short summary of the implemented changes in the commentary. You should consider editing your comment to add that, you can read more about it [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- [ ] According to the project requirements, your HTML and CSS file should contain the following:

![image](https://user-images.githubusercontent.com/66967274/148049739-6426debb-4579-4a5e-a690-bfac57075075.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


![so close](https://c.tenor.com/nMbdSNsg_jsAAAAC/the-simpsons-homer-simpson.gif)

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

zmekonnen251 commented:
Hello @VanessaAoki, as you suggested to me I have corrected all mistakes that I made earlier. I added a "Hello Microverse" heading in the HTML file and I styled it in the CSS file. I added the linter too in the local and remote repository.

elmar8287 commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,62 @@
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  eslint:
+    name: ESLint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
```
- [ ] Please, consider that you need to set up a linter only for HTML and CSS. Please, refer this [link](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml)
<img width="863" alt="Screen Shot 2022-01-04 at 23 22 58" src="https://user-images.githubusercontent.com/49064106/148112845-665aec14-2679-4dfc-ace9-b9287ee8f7a3.png">

zmekonnen251 replied:
Thank you @elmar8287  for your suggestion.

elmar8287 commented on style.css:
```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: red;
+  text-align: center;
+}
```
- [ ] Please make sure that you are using the class with styles for your `h1` instead of targeting it directly over the tag name.

zmekonnen251 replied:
Thank you for your suggestion.

elmar8287 reviewed (changes requested):
Hi @zmekonnen251 , 👋🏼  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

zmekonnen251 commented:
I have corrected all comments that I received from my code reviewers. I used HTML class to style an element instead of directly targeting the element itself. And another change that I made is, I previously used JavaScript linter but I removed the JavaScript linter now.

mmsesay reviewed (approved):
Hi @zmekonnen251 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

zmekonnen251 commented:
Thank you very much. All of you helped me a lot to know the GitHub flow and PR.
